tsconfig.json

```diff
@@ -65,11 +10,15 @@
     "test_utils/**/*"
   ],
   "exclude": [
-    "src/**/__fixtures__/**/*"
+    "src/**/__fixtures__/**/*",
+    "src/test_utils/**/*"
     // In the build we actually exclude **/public/**/* from this config so that
     // we can run the TSC on both this and the .browser version of this config
     // file, but if we did it during development IDEs would not be able to find
     // the tsconfig.json file for public files correctly.
     // "src/**/public/**/*"
+  ],
+  "references": [
+    { "path": "./src/test_utils/tsconfig.json" }
```

Comment:
nit: I believe the `/tsconfig.json` is unnecessary

Reply:
Navigation by click on the path doesn't work without the full format